mappings/net/minecraft/block/Block.mapping

```diff
@@ -436,12 +446,19 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 		FIELD field_10669 hardness F
 		FIELD field_10670 dynamicBounds Z
 		FIELD field_20721 opaque Z
+		FIELD field_21209 slowDownMultiplier F
+		FIELD field_21210 jumpVelocityMultiplier F
 		METHOD <init> (Lnet/minecraft/class_3614;Lnet/minecraft/class_3620;)V
+			ARG 1 material
 			ARG 2 materialColor
 		METHOD method_16228 dropsLike (Lnet/minecraft/class_2248;)Lnet/minecraft/class_2248$class_2251;
 			ARG 1 source
 		METHOD method_16229 dropsNothing ()Lnet/minecraft/class_2248$class_2251;
 		METHOD method_22488 nonOpaque ()Lnet/minecraft/class_2248$class_2251;
+		METHOD method_23351 velocityMultiplier (F)Lnet/minecraft/class_2248$class_2251;
+			ARG 1 velocityMultiplier
```

Comment:
Maybe just `multiplier` in the argument is enough here? (same with jumpVelocity)

Reply:
Hmm, I thought we tend to have the parameter names same as the method names for builder methods